packages/lib/CalendarService.ts

```diff
@@ -47,6 +47,17 @@ const getDuration = (start: string, end: string): DurationObject => ({
   minutes: dayjs(end).diff(dayjs(start), "minute"),
 });
 
+const buildUtcOffset = (minutes: number): string => {
+  const h =
+    minutes > 0
+      ? "+" + (Math.floor(minutes / 60) < 10 ? "0" + Math.floor(minutes / 60) : Math.floor(minutes / 60))
+      : "-" +
+        (Math.ceil(minutes / 60) > -10 ? "0" + Math.ceil(minutes / 60) * -1 : Math.ceil(minutes / 60) * -1);
+  const m = minutes > 0 ? minutes % 60 : (minutes % 60) * -1;
+  const offset = `${h}:${m}`;
+  return offset;
+};
```

Comment:
Simply takes in the offset in minutes and returns the offset in the format of `+-hh:mm`. This is done to satisfy the needs of our beloved ICAL parser to adjust the timezone of the event's start and end date and to avoid the GMT shift caused in the absence of `vTimezone`